tests/BrowserKitConfig.php

```diff
@@ -35,4 +35,29 @@ protected function supportsJs()
     {
         return false;
     }
+
+    public function skipMessage($testCase, $test)
+    {
+        if ('Behat\Mink\Tests\Driver\Form\Html5Test' === $testCase && 'testHtml5FormAction' === $test && version_compare($this->getDomCrawlerVersion(), '3.3.0.0-dev', '<')) {
+            return 'Symfony DomCrawler < 3.3 does not support the formAction attribute.';
+        }
+
+        return parent::skipMessage($testCase, $test);
+    }
+
+    private function getDomCrawlerVersion()
+    {
+        $installedFile = __DIR__.'/../vendor/composer/installed.json';
```

Comment:
btw, if we were to go for production-ready code, I would probably try to feature-detect the DomCrawler version rather than dealing with all possible edge cases to read the version from composer (and then having to account for the case where the install is not done through composer...)